sdk/ai/azure-ai-projects/samples/evaluations/sample_evaluations_schedules.py

```diff
@@ -1,81 +1,191 @@
-from azure.ai.projects import AIProjectClient
-
-from azure.identity import DefaultAzureCredential
-from azure.ai.projects.models import (
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+Installation Instructions
+
+pip install azure-identity azure-ai-projects azure-ai-ml
+
+# Optionally, you can `pip install azure-ai-evaluation` if you want a code-first experience to fetch evaluator id for built-in evaluators in code
+# refer https://learn.microsoft.com/en-us/azure/ai-studio/how-to/online-evaluation for more details
+"""
+import pprint
+from azure.ai.projects import AIProjectClient 
+from azure.identity import DefaultAzureCredential 
+from azure.ai.projects.models import ( 
     ApplicationInsightsConfiguration,
     EvaluatorConfiguration,
     EvaluationSchedule,
-    CronTrigger,
     RecurrenceTrigger,
-    Frequency,
-    RecurrenceSchedule,
+    Frequency
+)
+from azure.ai.evaluation import CoherenceEvaluator  
+
+# This sample includes the setup for an online evaluation schedule using the Azure AI Project SDK and Azure AI Evaluation SDK
+# The schedule is configured to run daily over the collected trace data while running two evaluators: CoherenceEvaluator and RelevanceEvaluator
+# This sample can be modified to fit your application's requirements
+
+# Name of your online evaluation schedule
+SAMPLE_NAME = "online_eval_name"
+
+# Name of your generative AI application (will be available in trace data in Application Insights)
+SERVICE_NAME = "service_name"
+
+# Connection string to your Azure AI Foundry project
+# Currently, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+# You can copy this connection string in the Azure AI Foundry portal > Project Overview > Project details > Project connection string
+PROJECT_CONNECTION_STRING = "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+
+# Your Application Insights resource ID
+APPLICATION_INSIGHTS_RESOURCE_ID = "appinsights_resource_id"
+
+# Kusto Query Language (KQL) query to query data from Application Insights resource
+# This query is compatible with data logged by the Azure AI Inferencing Tracing SDK (linked in documentation)
```

Comment:
I don't know what "Azure AI Inferencing Tracking SDK". Do you mean tracing created while doing chat completions operation using the azure-ai-inference SDK? (https://pypi.org/project/azure-ai-inference/) Please make it clear.

Reply:
yes, we were referring to azure-ai-inference SDK, updated code to make it clear.